scrapy/squeues.py

```diff
@@ -85,26 +87,56 @@ def _pickle_serialize(obj):
     # Both pickle.PicklingError and AttributeError can be raised by pickle.dump(s)
     # TypeError is raised from parsel.Selector
     except (pickle.PicklingError, AttributeError, TypeError) as e:
-        raise ValueError(str(e)) from e
+        raise SerializationError(str(e)) from e
+
+
+def _chunked_file_queue(queue_class):
+    class ChunckedFileQueue(queue_class):
+
+        def __init__(self, crawler, key, *args, **kwargs):
+            chunksize = crawler.settings.getint(
+                'SCHEDULER_DISK_QUEUE_CHUNKSIZE',
+                100000,
```

Comment:
We have this value defined in 3 places now: 

* here
* in default_settings
* in queuelib: https://github.com/scrapy/queuelib/blob/dc078fd3dfafafcf683dda13818664418d8e726e/queuelib/queue.py#L49

It could be good to have a single default value. What's the use case of changing queuelib's default?